zio-http/src/test/scala/zhttp/html/DomSpec.scala

```diff
@@ -62,22 +62,32 @@ object DomSpec extends DefaultRunnableSpec {
           Dom.text("zio-http"),
         )
 
-        assert(dom.encode)(
-          equalTo("""<a href="https://www.zio-http.com">zio-http</a>"""),
-        )
+        assertTrue(dom.encode == """<a href="https://www.zio-http.com">zio-http</a>""")
       } +
       suite("doctype") {
         test("empty5") {
           val dom = Dom.element("html")
-          assert(dom.encode)(equalTo("""<!DOCTYPE html><html/>"""))
+          assertTrue(dom.encode == """<!DOCTYPE html><html></html>""")
         } +
           test("with children") {
             val dom = Dom.element("html", Dom.element("head"))
-            assert(dom.encode)(equalTo("""<!DOCTYPE html><html><head/></html>"""))
+            assertTrue(dom.encode == """<!DOCTYPE html><html><head></head></html>""")
           } +
           test("with children and text") {
             val dom = Dom.element("html", Dom.element("head"), Dom.text("abc"))
-            assert(dom.encode)(equalTo("""<!DOCTYPE html><html><head/>abc</html>"""))
+            assertTrue(dom.encode == """<!DOCTYPE html><html><head></head>abc</html>""")
+          }
+      } + suite("VoidElements") {
```

Comment:
`Nitpicking` :-p please move the suite to next line

Reply:
Done.